### DISCUSSION TITLE
Does Room Assistant work on a Debian?

### DISCUSSION THREAD

**Discussion by shakarum** [mKeRix/room-assistant#440] [General]
Hi

I have my HA installed on a laptop with Debian 10.
Can Room Assistant run on this platform?

**Comment by mKeRix** [comment]
It should work fine on Debian, given that you have the hardware requirements for the integrations that you want to use (e.g. Bluetooth adapter/stick for Bluetooth integrations).

**Reply by shakarum** [reply]
Thanks. And which installation guide should I follow? Rpi? Docker?

**Reply by mKeRix** [reply]
Up to you - if you already have Docker installed that might be the easier path.

**Reply by shakarum** [reply]
I don't use Docker, it's just a pure Debian 10 that runs HA Supervised on a dedicated laptop.
